Index: monero-gui/monero-wallet-gui.pro
===================================================================
--- monero-gui.orig/monero-wallet-gui.pro
+++ monero-gui/monero-wallet-gui.pro
@@ -10,6 +10,7 @@ CONFIG += c++11
 QMAKE_DISTCLEAN += -r $$WALLET_ROOT
 
 INCLUDEPATH +=  $$WALLET_ROOT/include \
+		/usr/local/include/monero \
                 $$PWD/src/libwalletqt \
                 $$PWD/src/QR-Code-generator \
                 $$PWD/src \
@@ -88,6 +89,7 @@ ios:arm64 {
 }
 !ios {
 LIBS += -L$$WALLET_ROOT/lib \
+	-L/usr/lib/monero \
         -lwallet_merged \
         -lepee \
         -lunbound \
